### ISSUE TITLE
apache-http incorrect dependency

### ISSUE THREAD

**Issue by guillemsola** [chocolatey/chocolatey-coreteampackages#877]
I just installed apache-http with DSC extension on several azure machines with windows server 2016.

http.exe was complaining about a missing dll. I have seen that the package depends on the MSVCR 2015 but it should be 2012.

```
<dependency id="vcredist2015" />
```

As a workaround I installed the 2012 and apache installed succesfully.

**Comment by guillemsola** [comment]
I've created the pull req https://github.com/chocolatey/chocolatey-coreteampackages/pull/878

**Comment by AdmiringWorm** [comment]
actually, it's not the dependency that is incorrect. But rather the archive that was grabbed during the latest update run was the incorrect one.

I'm in the process of updating the update.ps1 script so it will grab the correct one, when I have I'll get a fixed version pushed out to chocolatey

**Comment by AdmiringWorm** [comment]
Just scheduled appveyor to push out a fixed release of the apache-httpd package.

as such I'm closing this one and the PR you opened, thank you for bringing this up to our attention though.

**Comment by guillemsola** [comment]
Ok so that's why appeared as installed the MSVC2017 instead I guess

**Comment by AdmiringWorm** [comment]
yup, that would be correct

**Comment by guillemsola** [comment]
Sorry but the package is still not fixed in chocolatey package. Currently I guess there is an issue with the vcredist2015 that installs the same package than vcredist2017. Although both should be api compatible I don't know if this is preventing apache from working.

**Comment by AdmiringWorm** [comment]
@guillemsola 
vcredist2017 should be fully backwards compatible with vcredist2015.
Which is why the meta package for vcredist2015 install the 2017 edition,
normally when installing the 2017 edition, it replaced the 2015 one.

What you can try and see if it works for you, is to first remove the 2017 redistributable (both manually, and from the package list), then install [vcredist140 version 14.0.24215.1][vcredist140] before installing apache-httpd.

[vcredist140](https://chocolatey.org/packages/vcredist140/14.0.24215.1)

**Comment by guillemsola** [comment]
That's why I'm afraid that vcredist2015 is not the ritght dependency for apache-httpd. What works for me now is installing vcredist2012 (as it used to be a few releases ago) before installing apache. 

With the default package on a brand new created azure 2015 server apache service is unable to work complaining of a missing vcredist dll.

**Comment by AdmiringWorm** [comment]
Just to confirm, you did try to install version `2.4.28.20171020`?
If that version doesn't work with vcredist2015, then that seems like a mistake from the developers as that version includes the vc14 version of the software, unlike the previous version that you reported on.

**Comment by guillemsola** [comment]
I'm creating a machine definition with little on it other than the Apache version you are mentioning

![image](https://user-images.githubusercontent.com/610476/31951837-687dcb22-b8df-11e7-9636-30809af92c5f.png)

**Comment by AdmiringWorm** [comment]
Hmm, then I'm afraid I actually don't really know.
Having vcredist2012 as the dependency would be wrong for that version, but I'll reopen the issue so we can investigate what is going on.

**Comment by guillemsola** [comment]
@AdmiringWorm I'm afraid the machine I was testing the package play a cache trick or so with me. I have installed on a brand new machine right now and package is correct. I also noticed that the checksum for the file is the correct one while the previous package corresponded to the httpd VC11 release.

We can safely close this ticket as everything is working as expected. 

Thanks for your support!
